src/core/util.ts

```diff
@@ -95,7 +95,7 @@ export function runBeforeNotifyHandlers(notice: Notice | null, handlers: BeforeN
 
 export function runAfterNotifyHandlers(notice: Notice | null, handlers: AfterNotifyHandler[], error?: Error): boolean {
   if (notice && notice.afterNotify) {
-    handlers.unshift(notice.afterNotify)
+    notice.afterNotify(error, notice)
```

Comment:
@joshuap This is a **bug fix** for a bug I introduced. When calling `runAfterNotifyHandlers` we are sending a reference to `this.__afterNotifyHandlers`, and at this line we are adding the notice's `afterNotify` handler. If a notice is to be sent before `hb.clear()` is called, this notice's `afterNotify` handler is still in `this.__afterNotifyHandler`. As a result, that handler is called twice.

I think we should release a patch for v3, but I'm confused where to target the PR. Changelog has `v3.2.8` as latest version, but I can't find a branch for it.

How I introduced this bug? I did it with [this commit](https://github.com/honeybadger-io/honeybadger-js/commit/71b38f23bb86df1a346a2aef7cae7ead70cc39d1#diff-e84e938a5120632cd9bfda48f378416139afdb46dbb35444370bae86a6f1c248L98) a couple of months ago as part of an effort to cleanup the code.

Reply:
@subzero10 here's the branch! Can you submit a PR there after this is merged? https://github.com/honeybadger-io/honeybadger-js/tree/v3-stable

Reply:
@joshuap False alarm, this bug was introduced with #659 , which is not on `v3`. We should release a new version on `v4` since this PR is now merged and it will be OK.